README.md

```diff
@@ -30,3 +30,10 @@ if your dev installation is older than 2015-05-15 you will need to add the initi
 php app/console do:mi:version --add 20150515104913
 ```
 
+running development machine
```

Comment:
I might change this later on, since not everyone wants to setup a large virtual machine for such simple projects

``` suggestion
development setup (vagrant)
```